wordcloud/wordcloud.py

```diff
@@ -6,6 +6,11 @@
 #
 # License: MIT
 
+import nltk
```

Comment:
Don't use nltk stopwords. Make a separate constant like stopwords.py with some stopword lists stored in it.